superbuild/projects_modules/TTK.cmake

```diff
@@ -39,8 +39,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Set up versioning control.
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/TTK-Public.git)
```

Comment:
For now, let's keep the architecture of repos. Once everything merged, move back to inria asclepios master